python/rpdk/java/codegen.py

```diff
@@ -66,19 +86,57 @@ def _prompt_for_namespace(self, project):
         project.settings["namespace"] = self.namespace
         self.package_name = ".".join(self.namespace)
 
-    def init(self, project):
-        LOG.debug("Init started")
+    @staticmethod
+    def _prompt_for_codegen_model(project):
+        prompt = "Choose codegen model - 1 (default) or 2 (guided-aws): "
 
-        self._prompt_for_namespace(project)
+        codegen_model = input_with_validation(
```

Comment:
any reason we just cant return the value that we need the settings key to be set to here:

```
        project.settings["codegen_template_path"] = input_with_validation(...)
```